understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
-      <figure id="figure-CSS-pixel-letter-4x"><img src="img/css-pixel-by-device.png" width="462" height="198" alt="Diagram showing the size of character needed by viewing distance to make the same image on the retina with small screen devices close, large screen devices further away."/>
-        <figcaption>A letter of the same CSS pixel size on different displays with different resolutions</figcaption>
-      </figure>
-        <p>When we read, the size of the print is not as important as the image it projects on the retina of our eye. Phones are designed for close viewing while desktops are designed for viewing farther away. As a consequence 16px print on a phone is physically smaller than 16px print on a desktop. This is not a problem because both print sizes cast the same image on our retina if they are viewed at their intended distance.</p>
-      </section>
-      <section>
-      <h3>Visibility and availability of content</h3> 
-      <p>How much of the content is visible may change at different scales. For example, navigation menus that are fully visible in the desktop layout are often collapsed into fewer items, or even into a single menu button (the 'hamburger' icon pattern) so they take up less screen space.</p>
-      <p>The Success Criterion is met as long as all content and functionality are still fully available -  either directly, or revealed via accessible controls, or accessible via direct links.</p>
-      </section>
+        <p>The intent of this success criterion is to let users enlarge (zoom in) multiline sections of content without having to perform bi-directional scrolling to read. Lines that extend beyond the viewport in the direction of reading require scrolling back-and-forth to read, which can significantly increase both physical and cognitive effort and cause users to lose their place. Outside of content which necessitates bi-directional layout to understand or function, content is expected to reflow within the appropriate sizing requirement defined by this success criteria, in regards to the content's intended direction of reading.</p>
+
+        <p>Reflow applies to both horizontally and vertically written languages. For pages where the primary written language expects vertical scrolling (such as English), users do not expect to scroll back-and-forth horizontally to read that content. Similarly, users reading content in a written language/direction that expects horizontal scrolling (such as traditional Chinese and Japanese) do not have to scroll up-and-down vertically to read that content.</p>
+
+        <div class="note">
+          <p>Reflow does not prohibit web pages from presenting both horizontal and vertical scrollbars for individual sections of content, nor does it disallow the use of bi-directional scrollbars at the page (viewport) level, when the content has been enlarged.</p>
```

Comment:
I'm concerned that the last half of this essentially lets almost every case drive through a giant loophole. It appears to say that when you have zoomed in, bi-directional scrolling is not a fail.

Reply:
thanks @alastc - i see your point.  The intent of this was to call out that bi-directional scroll bars at a page level do not necessarily mean the web page fails, so long as the scroll bars are there because of an exception, or if scroll bars appear but the content of the page can still be viewed without having to actually scroll in two directions to read it.

i hope that reading the rest of the doc makes this clear in context - but i can work on rewording this a bit so as to not to give off the wrong impression.